examples/fabric-sync/bridge/include/FabricBridge.h

```diff
@@ -0,0 +1,161 @@
+            std::nullopt,          /* uniqueId */                                                                                  \
+            std::nullopt,          /* vendorName */                                                                                \
+            std::nullopt,          /* vendorId */                                                                                  \
+            std::nullopt,          /* productName */                                                                               \
+            std::nullopt,          /* productId */                                                                                 \
+            std::nullopt,          /* nodeLabel */                                                                                 \
+            std::nullopt,          /* hardwareVersion */                                                                           \
+            std::nullopt,          /* hardwareVersionString */                                                                     \
+            std::nullopt,          /* softwareVersion */                                                                           \
+            std::nullopt,          /* softwareVersionString */                                                                     \
+            std::nullopt           /* isIcd */                                                                                     \
+    }
+
+#define AdministratorCommissioningChanged_init_default                                                                             \
+    {                                                                                                                              \
+        ScopedNodeId_init_default, /* id */                                                                                        \
+            0,                     /* windowStatus */                                                                              \
+            std::nullopt,          /* openerFabricIndex */                                                                         \
+            std::nullopt           /* openerVendorId */                                                                            \
+    }
+
+struct SynchronizedDevice
+{
+    chip::ScopedNodeId id;
+
+    std::optional<std::string> uniqueId;
+    std::optional<std::string> vendorName;
+    std::optional<chip::VendorId> vendorId;
+    std::optional<std::string> productName;
+    std::optional<uint16_t> productId;
+    std::optional<std::string> nodeLabel;
+    std::optional<uint16_t> hardwareVersion;
+    std::optional<std::string> hardwareVersionString;
+    std::optional<uint32_t> softwareVersion;
+    std::optional<std::string> softwareVersionString;
+    std::optional<bool> isIcd;
+};
+
+struct AdministratorCommissioningChanged
+{
+    chip::ScopedNodeId id;
+    uint8_t windowStatus;
```

Comment:
Is there a reason why we don't just type this to the enum that it should be already?